docker-compose.yml

```diff
@@ -67,9 +17,10 @@ services:
     extends:
       file: docker-services.yml
       service: es
-  kibana:
+  # Monitoring
+  flower:
```

Comment:
Do we need flower in a simple docker compose?

Reply:
It's sometimes useful to be able to see what's running (for long-running tasks, or failed ones)